packages/shorebird_cli/lib/src/shorebird_build_mixin.dart

```diff
@@ -3,7 +3,36 @@ import 'dart:io';
 import 'package:mason_logger/mason_logger.dart';
 import 'package:shorebird_cli/src/command.dart';
 
+enum Arch {
+  arm64,
+  arm32,
+  x86,
+}
+
+class ArchMetadata {
+  const ArchMetadata({required this.path, required this.arch});
+
+  final String path;
+  final String arch;
+}
+
 mixin ShorebirdBuildMixin on ShorebirdCommand {
+  // TODO(felangel): extend to other platforms.
+  static const architectures = <Arch, ArchMetadata>{
+    Arch.arm64: ArchMetadata(
+      path: 'arm64-v8a',
```

Comment:
I feel like these are flavors of paths. This one is for the build system? for the storage bucket? etc.

Reply:
yeah need to figure out the right abstraction...I just did the simplest thing for now